modules/manage/pages/manage-xdcr/incoming-xdcr-replications.adoc

```diff
@@ -0,0 +1,63 @@
+= Incoming Replications
+:description: pass:q[Incoming replications details for a target cluster in XDCR are listed on the UI and can be retrieved using the REST API.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+When a replication is created, the target cluster is identified as such.
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+
+[#view-incoming-replications]
+== View Incoming Replications
+
+On the XDCR UI, if the cluster is a target for XDCR replications from remote clusters, the Incoming Replications section lists the incoming replications, grouped by the remote cluster name.
+Expand *Incoming Replications* to see the details of any incoming replications created on the remote cluster.
```

Comment:
Expand the source cluster information under Incoming Replications to see the details of the replications created on the remote cluster.  In the example picture below, the Incoming Replications show that there are two remote clusters, East and South that are replicating data to the cluster.  From East, there is 1 replication incoming.  From South, there are 2 replications incoming, and the details of those replications are shown in the picture.